### PR TITLE
Support `import.meta.url`

### PR DESCRIPTION
**Pull request by yKanon**
description: use `import.meta.url` when use esmodule. but its value is 'file://xxxx'. so, `globalDir` return `undefine`, and `path.relative` throw Error

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,21 +2,23 @@
...
+  const globalDir = pkgDir.sync(path.dirname(normalizedFilename));
+  const relativePath = path.relative(globalDir, normalizedFilename);
+  const pkg = require(path.join(globalDir, 'package.json'));
+  const localFile = resolveCwd.silent(path.join(pkg.name, relativePath));
+  const localNodeModules = path.join(process.cwd(), 'node_modules');
 
-	const filenameInLocalNodeModules = !path.relative(localNodeModules, filename).startsWith('..') &&
-		// On Windows, if `localNodeModules` and `filename` are on different partitions, `path.relative()` returns the value of `filename`, resulting in `filenameInLocalNodeModules` incorrectly becoming `true`.
-		path.parse(localNodeModules).root === path.parse(filename).root;
+  const filenameInLocalNodeModules = !path.relative(localNodeModules, normalizedFilename).startsWith('..') &&
```
Comment (sindresorhus) [inline index.js]: Please don't make unrelated changes (the indentation).

**Comment by sindresorhus** [comment]
This is not fixing anything as `import.meta.url` support was never promised. That being said, I'm ok with the improvement.

You also need to update the readme to mention support for `import.meta.url`.

File: index.js (hunk 1 above)
Comment (yKanon) [reply index.js]: sorry, my fault. now, my modifications follow the lint information

**Comment by yKanon** [comment]
> This is not fixing anything as `import.meta.url` support was never promised. That being said, I'm ok with the improvement.
> 
> You also need to update the readme to mention support for `import.meta.url`.

alright. So, does big brother means that it is a feature?
